### PR TITLE
Adding AMD SEV-SNP attestation Page

### PR DESCRIPTION
**Pull request by carlosbravoa**

### CODE REVIEW

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+After switching to one of the regions mentioned above, launch the instance by enabling a CPU flag under advanced CPU options. 
+
+You can do it using either the AWS EC2 console or AWS CLI.
+
+If using AWS EC2 Console, scroll down to the Advanced details section and within it, scroll down to AMD SEV-SNP selector and select Enabled.
+
+.. image:: launch-and-attest-amd-sev-snp-instances-images/enable-amd-sev-snp.png
+
+If using AWS CLI, add ``--cpu-options AmdSecSnp=Enabled`` to your command:
```
Comment (rpocase) [inline aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:37]: should this be `AmdSevSnp`?

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    --instance-type supported_instance_type \
+    --key-name key_pair_name \
+    --subnet-id my_subnet_id \
+    --cpu-options AmdSevSnp=enabled
+
+Once the machine is running, you can check if AMD SEV-SNP was enabled by looking at the instance details on the EC2 console or using the AWS CLI command ``ec2 describe-instances`` API. ``AmdSevSnp`` should be in the CpuOptions
+
+More information at: `Check AMD SEV-SNP support on Amazon EC2 instances (AWS Site) <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-work.html>`_
+
+Build the attestation tool (snp guest tools)
```
Comment (rpocase) [inline aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:52]: fyi, we'll be able to replace this with snap installation instructions at some point. currently blocked on [upstream support](https://forum.snapcraft.io/t/access-to-dev-cpu-0-msr-from-a-strict-snap/41228) for certain snap confinement options

**Review by rpocase — CHANGES_REQUESTED** [review]

File: aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst (hunk 2 above)
Comment (carlosbravoa) [reply aws/aws-how-to/instances/launch-and-attest-amd-sev-snp-instances.rst:52]: That is way better. We can update this doc as soon as we have the snap

**Review by rpocase — APPROVED** [review]
lgtm pending CI being happy

**Pull request merged**